### PR TITLE
docs: Add a guide to using state sync in full node operations

### PR DESCRIPTION
**Pull request by audtlr24**

### CODE REVIEW

File: .gitbook/4-validators/3-join-mainnet-testnet.md [hunk 1]
```diff
@@ -160,6 +160,18 @@ panacead tendermint unsafe-reset-all
 
 ## Run a Full Node
 
+### Before Running a Node
+Before running full node, it is highly recommended to use [state sync](#state-sync).
+
+It is possible to start from scratch but will take a significant amount of time and effort to catch up.
+To sync full node from scratch, it is important to note that you must start with V2.0.0 and proceed through different upgrades at proper block height.
```
Comment (youngjoon-lee) [inline .gitbook/4-validators/3-join-mainnet-testnet.md:167]:
```suggestion
To sync full node from scratch, it is important to note that you must start with v2.0.1 and proceed through several upgrades at proper block heights.
```
https://github.com/medibloc/panacea-mainnet/blob/master/panacea-3/v2.0.1/upgrade.md

**Review by youngjoon-lee — APPROVED** [review]
LGTM!

**Review by H4NLee — APPROVED** [review]
:+1:

**Review by gyuguen — APPROVED** [review]
LGTM

**Pull request merged**